portal-kernel/src/com/liferay/portal/kernel/dao/jdbc/AutoBatchPreparedStatementUtil.java

```diff
@@ -255,11 +258,11 @@ private ConcurrentNoBatchInvocationHandler(
 			super(connection, sql);
 		}
 
+		private static final PortalExecutorManager _portalExecutorManager =
```

Comment:
@JanisZhang Please keep it consistent with before -- no `static`.

Reply:
My bad. Force-pushed. Thanks!